docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[network-beaconing-framework]]
+== Network Beaconing
+
+This feature provides an early warning system for command and control beaconing activity. It monitors network traffic for indicators of compromise and provides analytics to add context to alerts and aid your threat hunting.
+
+[discrete]
+=== Deploy the package
+
+To deploy the network beaconing framework in your environment using the https://github.com/elastic/detection-rules/releases/tag/ML-Beaconing-20211216-1[release package], follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+
+The release package includes dashboards for monitoring beaconing activity in your environment. You can review signals via a Lens dashboard called Network beaconing.
+
+[role="screenshot"]
+image::images/beaconing-detection-1.png[]
+
+[discrete]
+=== Feature details
+
+This feature uses a {ref}/transforms.html[transform] to segment network data by host and process name, then runs scripted metric aggregations on the host-process name pairs. For a given time window, the scripted metric aggregation checks each partition for the following:
+
+* Signals repeating at regular intervals, accounting for minor variations in those intervals.
+* Low variation of bytes sent from source to destination.
+* Low variation of bytes sent from destination to source.
```

Comment:
@ajosh0504 @SourinPaul I am slightly unsure about the language we are using here and just want to make sure I understand. What do we mean when we say: 
> "to segment network data by host/process name..."

Do we mean to filter? 

> "the scripted metric aggregation checks each partition for the following..."

What's a partition?  

Overall, I just want to make sure the language is simplistic and clear here.

Reply:
By segment we mean that we are splitting the data by host and process name. I don't think filter would mean the same?

Partition here is each pair of host-process name. You could replace "partition" with "pair" if that words better.